springwolf-add-ons/springwolf-common-model-converters/src/main/java/io/github/springwolf/addons/common_model_converters/converters/monetaryamount/MonetaryAmountConverter.java

```diff
@@ -19,7 +19,11 @@ public Schema resolve(AnnotatedType type, ModelConverterContext context, Iterato
             Class<?> cls = javaType.getRawClass();
             if (javax.money.MonetaryAmount.class.isAssignableFrom(cls)) {
                 type = new AnnotatedType(MonetaryAmount.class).resolveAsRef(true);
-                return (chain.hasNext()) ? chain.next().resolve(type, context, chain).name(javaType.getRawClass().getName()) : null;
+                Schema<?> schema = (chain.hasNext()) ? chain.next().resolve(type, context, chain) : null;
```

Comment:
Should we have just one call for chaining in the code? I am a bit worried that the second call might be overlooked in the future and cause problems. On the other hand it is clear when something is changed and when not because all the manipulation is in one place.